swap/src/cli.rs

```diff
@@ -1,38 +1,74 @@
-use libp2p::core::Multiaddr;
+use libp2p::{core::Multiaddr, PeerId};
+use std::str::FromStr;
 use url::Url;
 use uuid::Uuid;
 
 #[derive(structopt::StructOpt, Debug)]
 #[structopt(name = "xmr-btc-swap", about = "Trustless XMR BTC swaps")]
 pub enum Options {
     Alice {
-        #[structopt(default_value = "http://127.0.0.1:8332", long = "bitcoind")]
+        #[structopt(
+            short = "b",
+            long = "bitcoind",
+            default_value = "http://127.0.0.1:8332"
+        )]
         bitcoind_url: Url,
 
-        #[structopt(default_value = "http://127.0.0.1:18083/json_rpc", long = "monerod")]
-        monerod_url: Url,
+        #[structopt(
+            short = "m",
+            long = "monero-wallet-rpc",
+            default_value = "http://127.0.0.1:18083/json_rpc"
+        )]
+        monero_wallet_rpc_url: Url,
 
-        #[structopt(default_value = "/ip4/127.0.0.1/tcp/9876", long = "listen-addr")]
+        #[structopt(
+            short = "a",
+            long = "listen-addr",
+            default_value = "/ip4/127.0.0.1/tcp/9876"
+        )]
         listen_addr: Multiaddr,
 
-        #[structopt(long = "tor-port")]
+        #[structopt(short = "t", long = "tor-port")]
         tor_port: Option<u16>,
+
+        #[structopt(short = "s", long = "send-piconeros", parse(try_from_str = parse_pics))]
+        send_monero: xmr_btc::monero::Amount,
+
+        #[structopt(short = "r", long = "receive-sats", parse(try_from_str = parse_sats))]
+        receive_bitcoin: bitcoin::Amount,
     },
     Bob {
-        #[structopt(long = "sats")]
-        satoshis: u64,
-
-        #[structopt(long = "alice-addr")]
+        // TODO: Remove - Seems this is not needed anymore...?
+        // #[structopt(long = "sats")]
+        // satoshis: u64,
+        #[structopt(short = "a", long = "alice-addr")]
         alice_addr: Multiaddr,
 
-        #[structopt(default_value = "http://127.0.0.1:8332", long = "bitcoind")]
+        #[structopt(short = "p", long = "alice-peer-id")]
+        alice_peer_id: PeerId,
+
+        #[structopt(
+            short = "b",
+            long = "bitcoind",
+            default_value = "http://127.0.0.1:8332"
+        )]
         bitcoind_url: Url,
 
-        #[structopt(default_value = "http://127.0.0.1:18083/json_rpc", long = "monerod")]
-        monerod_url: Url,
+        #[structopt(
+            short = "m",
+            long = "monerod",
+            default_value = "http://127.0.0.1:18083/json_rpc"
+        )]
+        monero_wallet_rpc_url: Url,
 
-        #[structopt(long = "tor")]
+        #[structopt(short = "t", long = "tor")]
```

Comment:
Does tor even work? I don't think it does.

Reply:
I don't care at the moment.